### PR TITLE
Update view-reports-for-atp.md

### PR DESCRIPTION
**Pull request by kellybos**
Need to clarify that filter option "Domains" refers to the URL domain from report. Not the sender/recipient email domain.

Thanks!

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/view-reports-for-atp.md [hunk 1]
```diff
@@ -313,6 +313,9 @@ If you click **View details table**, the report provides a near-real-time view o
 
 If you click **Filters** in the details table view, you can filter by the same criteria as in the report view, and also by **Domains** or **Recipients** separated by commas.
 
+[!NOTE]
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/view-reports-for-atp.md]:
```suggestion
> [!NOTE]
```

File: microsoft-365/security/office-365-security/view-reports-for-atp.md [hunk 2]
```diff
@@ -313,6 +313,9 @@ If you click **View details table**, the report provides a near-real-time view o
...
+The Domains filter refers to the URL domain listed found in the report results.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/view-reports-for-atp.md]:
```suggestion
> The **Domains** filter refers to the URL domain listed in the report results. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@kellybos

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@kellybos - Please also commit the first edit (> [!NOTE]). Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**